### DISCUSSION TITLE
"workflow was not found" when trying to call reusable workflow

### DISCUSSION THREAD

**Discussion by Piedone** [community/community#70251] [Actions] [answered]
### Select Topic Area

Question

### Body

I'm trying to call [this reusable workflow kept in a public repo](https://github.com/GovBuilt/GitHub-Actions/blob/522ce6f2b1d4a58f445b1b0b6e51abd4f7ee50ae/.github/workflows/build-and-test-orchard-core.yml) from a private repo of the same organization:

```yml
jobs:
  build-and-test:
    name: Build and Test
    uses: GovBuilt/GitHub-Actions/.github/workflows/build-and-test-orchard-core.yml@dev
```

I think this is supposed to work. However, I get the below error:

```
[Invalid workflow file: .github/workflows/build-and-test.yml
error parsing called workflow
".github/workflows/build-and-test.yml"
-> "GovBuilt/GitHub-Actions/.github/workflows/build-and-test-orchard-core.yml@dev"
: workflow was not found. See https://docs.github.com/actions/learn-github-actions/reusing-workflows#access-to-reusable-workflows for more information.
```

I did check out [the linked docs page](https://docs.github.com/en/actions/using-workflows/reusing-workflows#access-to-reusable-workflows) and I'm still none the wiser: AFAIK, all points check out. I'm not entirely sure which config "your organization allows you to use public reusable workflows" is referring to, but since even the most restrictive config under https://github.com/organizations/MyOrg/settings/actions allows workflows from the same organization, it shouldn't be an issue.

Calling the exact same workflow from [the public repo under a different org it was forked from](https://github.com/Lombiq/GitHub-Actions) works without issues.

Why is the one in the same org not working?

**Comment by Piedone** [comment, marked as answer]
I got a reply from GitHub support. The problem was that actions in the forked repository (i.e. actions to run on that repository) were not enabled; this is the config for forks by default. I enabled it under https://github.com/GovBuilt/GitHub-Actions/actions and now the build works.

**Reply by ShoshanaMichel** [reply]
Please can someone tell me how can I get paid GitHub support.  I get an automated email from them, but no response.

**Reply by Piedone** [reply]
I got in touch with the standard GitHub support as a paying user on the Team plan, and got this resolve within a week or so.

**Comment by ShoshanaMichel** [comment]
Can you be more specific – where do you sign up to be a paying user on the team plan?

 

Shoshana Michel | Software Developer | UnbeatableSale, Inc.

**Reply by Piedone** [reply]
See https://github.com/pricing.

**Comment by ShoshanaMichel** [comment]
I had tried the tab for “Enterprise” that’s when I got Contact Sales.

We’re really looking for phone support, to get help setting up deployment actions.

In the Team Support I can open a case, and someone will respond in writing only, I guess?

 

Shoshana Michel | Software Developer | UnbeatableSale, Inc.

**Reply by Piedone** [reply]
Yes, but these are things that I'm afraid I can't really help you with, only GitHub can.

**Comment by ShoshanaMichel** [comment]
Ok – thank you.

 

Shoshana Michel | Software Developer | UnbeatableSale, Inc.
